### PR TITLE
Improve coding style and C# language usage.

### PR DESCRIPTION
**Pull request by EdiWang**
Use .NET and C# coding standard

### CODE REVIEW

**Comment by Anduin2017** [comment]
Reviewed. This pull request fixed all C# naming issues. Strongly suggest merging it.

File: ontology-csharp-sdk/Wallet/Account.cs [hunk 1]
```diff
@@ -4,9 +4,8 @@ namespace OntologyCSharpSDK.Wallet
 {
     public class Account
     {
-        
-        public string address = "";
-        public string enc_alg = "";
+        public string address { get; set; } = "";
```
Comment (Anduin2017) [inline ontology-csharp-sdk/Wallet/Account.cs]: I suggest to rename it to `Address` and give it default value as `string.Empty`

**Comment by EdiWang** [comment]
There are other coding standard issues, I will create another PR after this one.

**Comment by panther142** [comment]
> Use .NET and C# coding standard

Thanks for the PR - what is your association with Ontology? How did you start working on this repo?

File: ontology-csharp-sdk/Wallet/Account.cs (hunk 1 above)
Comment (EdiWang) [reply ontology-csharp-sdk/Wallet/Account.cs]: Agree, but I still need to do some research on these naming changes. I am not sure if camelCase is required in ONT API requests. If so, we need to figure out how to serialize PascalCase to camelCase just like the serialization options in .NET Core.

**Comment by panther142** [comment]
Hi @EdiWang.

Are you able to seperate the coding style changes from new classes and functionality?

This will make it easier to review and manage.

I have some uncommited changes that apply the majority of the coding style changes in your PR.

**Comment by EdiWang** [comment]
Hi @panther142 

These new classes are actually moved from inside "TransactionBuilder.cs", they are not newly added functionalities.

image

Do you need me to put them back in order to do a Diff view more clearly?

File: ontology-csharp-sdk/Wallet/Account.cs (hunk 1 above)
Comment (EdiWang) [reply ontology-csharp-sdk/Wallet/Account.cs]: Changed in 9c7b6c9

**Comment by panther142** [comment]
It's ok, i will merge shortly. If we can try and seperate out the PRs into smaller ones in the future that would be great. Thanks for your contribution.

**Pull request merged**